collections/_article/rendering-acyclic-graph.md

```diff
@@ -0,0 +1,277 @@
+---
+title: "GPU synchronization in Godot 4.3 is getting a major upgrade"
+excerpt: "Say goodbye to placing barriers, hello to the new acyclic graph"
+categories: ["progress-report"]
+author: Darío Banini
+image: /storage/blog/covers/todo.webp
+date: 2024-02-14 9:00:00
+---
+
+*Darío, the author of this article, is a new face around this blog who was hired by **W4 Games** last summer to start contributing to the project. His work was kindly sponsored and donated to the Godot Engine project by **W4 Games**. You can find more of his contributions to the engine in his [GitHub profile](https://github.com/DarioSamo).*
+
+Since the introduction of Godot 4, [Rendering Device](https://docs.godotengine.org/en/stable/classes/class_renderingdevice.html) has been the backbone of the Forward+ and Mobile renderers. Making APIs that are both easy to use and flexible enough to cover all the features users want in a game engine is a very difficult task. The high level of detail expected by APIs like Vulkan or Direct3D 12 compared to their predecessors is worthy of a few blog posts of its own. This article will try to keep the topic as brief as possible to focus on the problems solved within Godot instead.
```

Comment:
RenderingDevice is a class name, so it should be written as PascalCase:

```suggestion
Since the introduction of Godot 4, [RenderingDevice](https://docs.godotengine.org/en/stable/classes/class_renderingdevice.html) has been the backbone of the Forward+ and Mobile renderers. Making APIs that are both easy to use and flexible enough to cover all the features users want in a game engine is a very difficult task. The high level of detail expected by APIs like Vulkan or Direct3D 12 compared to their predecessors is worthy of a few blog posts of its own. This article will try to keep the topic as brief as possible to focus on the problems solved within Godot instead.
```